service/history/historyEngine2_test.go

```diff
@@ -299,6 +300,7 @@ func (s *engine2Suite) TestRecordDecisionTaskStartedSuccessStickyEnabled() {
 	s.NotNil(response)
 	expectedResponse.StartedTimestamp = response.StartedTimestamp
 	expectedResponse.ScheduledTimestamp = common.Int64Ptr(0)
+	response.ScheduledTimestamp = common.Int64Ptr(0)
```

Comment:
I realise this has a new clock construct in it, I just didn't want to refactor a huge amount of this suite test